fontc/src/lib.rs

```diff
@@ -502,7 +502,7 @@ fn add_os2_be_job(
     Ok(())
 }
 
-fn add_hmetrics_job(
+fn add_metric_and_limits_job(
```

Comment:
sometimes you use plural for both metrics and limits (as in the module name), other times singlular for both as in 
"create_metric_and_limit_work" function, othertimes singular for one and plural for the other; I know it doesn't matter much from a meaning point of view but it is a bit distracting. If you remove one item and only name "metrics", without the "_and_limits" there's one less thing to worry about. You can document that updating the metrics has side effect of also updating other relaated maximum limits.